src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -275,7 +276,8 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
       originLevel3: document.referrer,
       originContext: this.originContext,
       customData: _.keys(metaObject).length > 0 ? metaObject : undefined,
-      userAgent: navigator.userAgent
+      userAgent: navigator.userAgent,
+      clientId: new AnalyticsInformation().clientId
```

Comment:
Reading the `clientId` no longer initializes a `clientId`. If analytics aren't enabled, this is going to be `null`.